utils/jmxfiles.py

```diff
@@ -0,0 +1,109 @@
+# std
+import logging
+import os
+import tempfile
+import time
+
+# 3rd party
+import yaml
+
+# datadog
+from config import _windows_commondata_path
+from util import yDumper
+from utils.pidfile import PidFile
+from utils.platform import Platform
+
+
+log = logging.getLogger(__name__)
+
+
+class JMXFiles(object):
```

Comment:
Yesah but if you use `from utils import jmxfiles` there can't be a conflict with any other module defining a method with the same name. Am I wrong ?